packages/lib-extensions/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@openshift/dynamic-plugin-sdk-extensions",
-  "version": "1.2.0",
+  "version": "1.3.0",
```

Comment:
By convention, we bump the minor semver number due to change in package dependencies.